src/shared/GlobalTopBanners/SentryTrialBanner/SentryTrialBanner.spec.tsx

```diff
@@ -6,6 +6,8 @@ import { setupServer } from 'msw/node'
 import { Suspense } from 'react'
 import { MemoryRouter, Route } from 'react-router-dom'
 
+import { TrialStatuses } from 'services/account'
```

Comment:
This SentryTrialBanner file is going to be removed/renamed soon iirc?

Reply:
Yes ... in a later PR I don't know if this is gonna cause a bit of a refactoring headache ...